src/python/ensembl/io/genomio/fasta/process.py

```diff
@@ -96,6 +99,13 @@ def main() -> None:
     parser.add_argument_src_path("--genbank_infile", help="Input GenBank GBFF file")
     parser.add_argument_dst_path("--fasta_outfile", required=True, help="Output FASTA file")
     parser.add_argument("--peptide_mode", action="store_true", help="Process proteins instead of DNA")
+    parser.add_log_arguments()
     args = parser.parse_args()
 
+    init_logging(
+        args.log_level,
+        log_file=args.log_file,
+        log_file_level=args.log_file_level,
+    )
+
     prep_fasta_data(**vars(args))
```

Comment:
I think you need to alter the input to prep_fasta_data as it will likely complain about the additional log arguments 
`prep_fasta_data(args.fasta_infile, args.genbank_infile, args.fasta_outfile, args.peptide_mode)`

Reply:
thank you!